src/org/labkey/test/tests/component/GridPanelTest.java

```diff
@@ -190,14 +192,14 @@ private void createSmallSampleType() throws IOException, CommandException
         log(String.format("Create the '%s' view for the '%s' sample type.", VIEW_FEWER_COLUMNS, SMALL_SAMPLE_TYPE));
         cv = drtSamples.openCustomizeGrid();
 
-        for(String columnName : removedColumns)
+        for(String columnName : List.of(FILTER_BOOL_COL.toString()))
         {
             cv.removeColumn(columnName);
         }
```

Comment:
CustomizeView supports FieldInfo/FieldKey now
```suggestion
        for(FieldInfo column : List.of(FILTER_BOOL_COL))
        {
            cv.removeColumn(column);
        }
```

Reply:
Done